Apps/System/music.c

```diff
@@ -0,0 +1,461 @@
+/* music.c
+ *
+ * Music player
+ *
+ * RebbleOS
+ * 
+ * Author: Chris Multhaupt <chris.multhaupt@gmail.com>.
+ */
+
+#include "rebbleos.h"
+#include "music.h"
+#include "property_animation.h"
+
+#define RECORD_CENTER_X 56
+#define RECORD_CENTER_Y 57
+#define RECORD_CENTER GPoint(RECORD_CENTER_X, RECORD_CENTER_Y)
+#define RECORD_SWOOSH_DISTANCE 130
+#define RECORD_SWOOSH_SPEED 250
+#define ARM_OFFSET_X 17
+#define ARM_OFFSET_Y 77
+#define ARM_OFFSET GPoint(ARM_OFFSET_X, ARM_OFFSET_Y)
+#define ARM_OFFSET_ANGLE -31
+#define ARM_HOME_ANGLE 0
+#define ARM_START_ANGLE 18
+#define ARM_END_ANGLE 46
+#define ARM_SKIP_SPEED 200
+#define ARM_QUICK_SPEED 100
+#define SKIP_DIRECTION_PREV -1
+#define SKIP_DIRECTION_NEXT 1
+#define PROGRESS_PIXELS_MAX 105
+
+static Window *s_main_window;
+static Layer *s_main_layer;
+static ActionBarLayer *s_action_bar;
+
+static n_GPath *s_arm_base_path_ptr;
+static n_GPath *s_arm_1_path_ptr;
+static n_GPath *s_arm_2_path_ptr;
+static n_GPath *s_arm_head_path_ptr;
+
+static const n_GPathInfo ARM_BASE_PATH_INFO = {
```

Comment:
Even if they're constant, I assume they take space in ROM; so I'd probably call this `pathinfo_arm_base` or something (and, notably, not all caps).